Multisig/UI/Settings/SafeSettingsViewModel.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  SafeSettingsViewModel.swift
+//  Multisig
+//
+//  Created by Moaaz on 5/6/20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+import Combine
+
+class SafeSettingsViewModel: ObservableObject {
+    
+    @Published
+    var isResolving: Bool?
+
+    @Published
+    var isValid: Bool?
+
+    @Published
+    var errorMessage: String = ""
+
+    @Published
+    var address: String = ""
+    
+    @Published
+    var info: SafeStatusRequest.Response?
+    
+    private var subscribers = Set<AnyCancellable>()
+
+    func resolve() {
```

Comment:
the name should be `load()`, because we're loading the safe info, resolving is not appropriate here.

same for the `startResolving()` method